staging/src/k8s.io/apimachinery/Godeps/Godeps.json

```diff
@@ -115,6 +143,126 @@
 			"Rev": "cca7078d478f8520f85629ad7c68962d31ed7682"
 		},
 		{
+			"ImportPath": "github.com/onsi/ginkgo",
```

Comment:
Any chance we can get rid of these depdencies? We want to keep apimachinery's dependency as small as possible.

Reply:
Yeah, I really don't have a strong feeling about `ginkgo`. Some people really like it, I'm usually also in favor of minimal dependencies.